tests/test_storageclass_deletion.py

```diff
@@ -0,0 +1,71 @@
+"""
+Automates the test OCS-297:-
+1) Create a Storage Class
+2) Create a PVC
+3) Delete the corresponding storage class
+"""
+import logging
+import pytest
+
+from ocsci.testlib import tier1, ManageTest
+from ocs import constants
+from tests import helpers
+import ocs.defaults as defaults
+
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='class')
+def test_fixture(request):
+    """
+    Deleting a Storage class
+    """
+    self = request.node.cls
+
+    def finalizer():
+        teardown(self)
+    request.addfinalizer(finalizer)
+    setup(self)
+
+
+def teardown(self):
+    """
+    Remove the resources after execution of tests
+    """
+
+
+def setup(self):
+    """
+    Setting up the Environment : Creating project
+    """
+    project_name = defaults.NAMESPACE
+    log.info("Creating project")
+    NAMESPACE.new_project(project_name)
+
+
+@tier1
+@pytest.mark.usefixtures(
+    test_fixture.__name__,
+)
+class TestStorageClass(ManageTest):
+
+    @tier1
+    def test_storage_class(self, request):
+        """
+        Test to validate OCS-297
+        """
+        project_name = defaults.NAMESPACE
+        global PVC, STORAGE_CLASS, NAMESPACE
+        log.info("Creating StorageClass")
+        STORAGE_CLASS = helpers.create_storage_class(
+            constants.CEPHBLOCKPOOL, 'rbd'
+        )
+        log.info("Creating a PVC")
+        PVC = helpers.create_pvc(STORAGE_CLASS.name)
+        log.info("Deleting PVC")
+        PVC.delete()
```

Comment:
Why do you delete PVC right after its creation? 
Test steps of OCS-297 state that only the storage class should be deleted but the PVC should remain intact and fucntional.

Reply:
The main purpose of the testcase was to see the created storage class worked or not. SO creating a PVC verifies it. Hence immediatly deleting the PVC after creation.

Reply:
This is not part of the test case in Polarion

Reply:
removed and added it tear down